build_library/catalyst_toolchains.sh

```diff
@@ -16,10 +16,10 @@ configure_target_root() {
     local cross_chost=$(get_board_chost "$1")
     local profile=$(get_board_profile "${board}")
 
-    CHOST="${cross_chost}" \
-        ROOT="/build/${board}" \
-        SYSROOT="/usr/${cross_chost}" \
-        _configure_sysroot "${profile}"
+    CBUILD="$(portageq envvar CBUILD)" \
+        CHOST="${cross_chost}" \
+        ROOT="$ROOT" SYSROOT="$ROOT" \
+        _configure_sysroot "${CROSS_PROFILES[${cross_chost}]}"
```

Comment:
Why are you switching from the board profile  to to the generic CROSS_PROFILES? Does your board profile differ from the cross profile? Does your board profile not work for building a native toolchain?

Worthy of note on this stuff: In the case of amd64 we build the native toolchain with the intent to create developer images/containers that have a full native dev environment with emerge and everything. That goal may not be practical for other architectures, in which case providing glibc and gcc's libstdc++.so/libgcc_s.so by other means may be reasonable. ChromiumOS handled this by just copying the cross compiler's libs instead of building a full toolchain like I do now.